VSRAD.Package/DebugVisualizer/SliceVisualizer/SliceRowStyling.cs

```diff
@@ -10,10 +10,16 @@ sealed class SliceRowStyling
         public SliceRowStyling(SliceVisualizerTable table)
         {
             _table = table;
-            _table.RowPostPaint += ReplaceDefaultRowHeaderBitmap;
+            
         }
 
-        private void ReplaceDefaultRowHeaderBitmap(object sender, DataGridViewRowPostPaintEventArgs e)
+        public static void ApplyRowStylingOnRowPostPaint(SliceVisualizerTable table)
```

Comment:
I think you can remove the phrase RowStyling from the method name as it would be used by lines like `SliceRowStyling.ApplyOnRowPostPaint` anyway.
Also, you can make ReplaceDefaultRowHeaderBitmap and constructor private if they are only supposed to be called from within the class.

Reply:
Thanks for the suggestion, resolved in addbe4a